### PR TITLE
docs: add office hour 23062023

### PR DESCRIPTION
**Pull request by carslen**
Add recent office hour to our website.

### CODE REVIEW

**Review by almadigabor — APPROVED** [review]

File: blog/2023-06-23-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  a [room for Eclipse-Tractus-X](https://matrix.to/#/#tools.tractus-x:matrix.eclipse.org).
+- Next Catena-X will hit holiday season in August. Keep in mind, that resources at Eclipse Foundation (IP-Team),
+  System-Team and Security-Team will have only limited resources available during holiday season.
+- Repositories in Eclipse-Tractus-X should not stay for a long time in an uninitialized state, or without reasonable
+  content. Take care of your repositories.
+
+## Open discussions
+
+- Problems with ECA checks and issues to rewrite the Git history with the correct email-address. Issue will be clarified
+  in a dedicated 101 session.
```
Comment (SebastianBezold) [inline blog/2023-06-23-OfficeHours.md:39]: Are we actually scheduling an extra '101' session? Or do you mean one-on-one meeting?

Comment (carslen) [reply blog/2023-06-23-OfficeHours.md:39]: That's what it was meant. Looking forward if he will get in contact with us.

Comment (SebastianBezold) [reply blog/2023-06-23-OfficeHours.md:39]: Ok so then I would suggest to call it one-on-one. 101 does give me the impression, that we either send out invitations or record some tips and tricks video

Comment (carslen) [reply blog/2023-06-23-OfficeHours.md]: adjusted wording.

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**